### PR TITLE
New package: aome510.spotify-player version 0.17.1

### PR DESCRIPTION
**Pull request by gbostock**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146116)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @250h,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 873.)

**Comment by 250h** [comment]
It was identifies as infected by Windows Defender, but for the first time passed. Windows Defender always misidentifies programs written in Rust as viruses.

**Comment by stephengillie** [comment]
Hi @250h,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 873.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-03-25 13:29:02.566 [FAIL] Installer failed security check. Url: https://github.com/aome510/spotify-player/releases/download/v0.17.1/spotify_player-x86_64-pc-windows-msvc.zip Result: 0x80004005

(Automated response - build 873.)

**Comment by stephengillie** [comment]
Hi @250h,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-03-26 08:30:27.355 [FAIL] Installer failed security check. Url: https://github.com/aome510/spotify-player/releases/download/v0.17.1/spotify_player-x86_64-pc-windows-msvc.zip Result: 0x80004005

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Hi @250h,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-03-27 09:29:17.004 [FAIL] Installer failed security check. Url: https://github.com/aome510/spotify-player/releases/download/v0.17.1/spotify_player-x86_64-pc-windows-msvc.zip Result: 0x80004005

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\aome510.spotify-player__DefaultSource\spotify_player.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\spotify_player.exe returned exit code: -1073741515

(Automated response - build 876.)

File: manifests/a/aome510/spotify-player/0.17.1/aome510.spotify-player.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: aome510.spotify-player
+PackageVersion: 0.17.1
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: spotify_player.exe
+  PortableCommandAlias: spotify_player
+ReleaseDate: 2024-03-13
+Installers:
```
Comment (stephengillie) [inline manifests/a/aome510/spotify-player/0.17.1/aome510.spotify-player.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 876.)

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify: 
image

**Pull request merged**